consensus/core/src/block.rs

```diff
@@ -112,6 +139,24 @@ impl BlockRef {
             digest,
         }
     }
+
+    pub fn round(&self) -> Round {
+        self.round
+    }
+
+    pub fn author(&self) -> AuthorityIndex {
+        self.author
+    }
+
+    pub fn digest(&self) -> BlockDigest {
+        self.digest
+    }
+}
+
+impl From<BlockRef> for AuthorityRound {
```

Comment:
What is the advantage of using AuthorityRound instead of (AuthorityIndex, Round)?